zone/zonedb.cpp

```diff
@@ -2728,6 +2729,7 @@ const NPCType *ZoneDatabase::LoadNPCTypesData(uint32 npc_type_id, bool bulk_load
 		temp_npctype_data->use_model        	= atoi(row[110]);
 		temp_npctype_data->flymode          	= atoi(row[111]);
 		temp_npctype_data->always_aggro	        = atoi(row[112]);
+		temp_npctype_data->can_open_doors		= atoi(row[113]);
```

Comment:
Should return a bool here just to be on the safe side.
```cpp
temp_npctype_data->can_open_doors		= atoi(row[113]) != 0 ? true : false;

Reply:
Just need the `!= 0` bit, it's already a bool with just that. (We do that in a lot of places)

Reply:
Done.   Fixed my previous field as well.

Reply:
👍